WebKit.xcworkspace/contents.xcworkspacedata

```diff
@@ -40,6 +40,9 @@
    <FileRef
       location = "group:Source/ThirdParty/gtest/xcode/gtest.xcodeproj">
    </FileRef>
+   <FileRef
+      location = "container:Source/ThirdParty/gmock/gmock.xcodeproj">
```

Comment:
I don't know the difference between "group:" and "container:" in this format, but it does strike me that this one is different from all the rest. And I suspect we don't actually need gmock in the workspace, because it's already available through nested projects, and Apple's internal build builds individual projects and not schemes in the workspace.

Reply:
It's actually not available through nested projects, at the moment. I did this through the UI, I'll change this to match everything else.